README.md

```diff
@@ -89,6 +90,8 @@ This is very useful to inspect pipeline state during debugging, for example.
 - Because everything is just lines with separate expressions and not one huge function call, IDEs can show exactly in which line errors happened
 - Pipe is a name defined by Base Julia which can lead to conflicts
 
+For more details, see the [pre-announcement post](https://discourse.julialang.org/t/rfc-pipelesspipes-jl-now-chain-jl/50448).
```

Comment:
I don't really think this is necessary, the readme has all the information needed and that thread was mostly about deciding the last details

Reply:
Sure, I can remove it. I personally find it interesting to be able to follow the early history of projects, but I understand it can add more noise than actual useful information.